examples/barotropicqg_betaforced.jl

```diff
@@ -53,26 +53,27 @@ forcing_wavenumber = 14.0    # the central forcing wavenumber for a spectrum tha
 forcing_bandwidth  = 1.5     # the width of the forcing spectrum 
 ε = 0.001                    # energy input rate by the forcing
 
-gr  = TwoDGrid(nx, Lx)
+gr  = TwoDGrid(n, L)
```

Comment:
Is this necessary? I noticed the betadecay example doesn't explicitly define the grid, so is there a reason the betaforced example does?

Reply:
Because wave numbers are needed to construct the forcing spectrum below. I know it’s not neat but I can’t think of a better way.

Reply:
Ah I see. The decay example pulls the grid from the initialized problem, but the forced example needs the grid in order to initialize the problem with that spectrum. Thanks!